.github/workflows/tools/containers/buildenv-git-annex-buster/Dockerfile

```diff
@@ -8,13 +8,18 @@ RUN echo 'Acquire::http::Dl-Limit "1000";' >| /etc/apt/apt.conf.d/20snapshots \
     && echo 'Acquire::https::Dl-Limit "1000";' >> /etc/apt/apt.conf.d/20snapshots \
     && echo 'Acquire::Retries "5";' >> /etc/apt/apt.conf.d/20snapshots
 
+# We need to enable backports to get more recent than 2.30 git.
+# ref: https://git-annex.branchable.com/forum/error__58___bogus_format_in_GIT__95__CONFIG__95__PARAMETERS/
 RUN set -ex; \
-    apt update; apt install neurodebian-freeze; \
+    apt update; \
```

Comment:
Shouldn't you do `apt update` *after* editing the sources list?

Reply:
doh -- in principle - right and I will move update down.  in practice here doesn't matter since nd_freeze would update too, and prior update would be sufficient to install neurodebian-freeze (does not come from backports).